batch/batch/worker/worker.py

```diff
@@ -1316,6 +1588,8 @@ def __init__(self):
         self.task_manager = aiotools.BackgroundTaskManager()
         self.jar_download_locks = defaultdict(asyncio.Lock)
 
+        self.bundle_locks = defaultdict(asyncio.Lock)
```

Comment:
bundle is a bit of a surprising name. Is that the OCI terminology for an image?

Reply:
Missed this when renaming bundles to `rootfs`. In OCI jargon, they consider a bundle to be a directory of the rootfs and the container's config.json (i.e. everything you need to run a container). Thankfully, crun allows us to store the config.json separately from the rootfs, so there's not much need for the term "bundle".